inspectors/models/notation/notes/beams/beamsettingsmodel.cpp

```diff
@@ -197,7 +195,7 @@ void BeamSettingsModel::setBeamModesModel(BeamModesModel* beamModesModel)
 
     connect(m_beamModesModel->isFeatheringAvailable(), &PropertyItem::propertyModified, this, [this] (const QVariant& newValue) {
         if (!newValue.toBool()) {
-            setFeatheringMode(static_cast<int>(BeamTypes::FEATHERING_NONE));
+            setFeatheringMode(BeamTypes::FeatheringMode::FEATHERING_NONE);
```

Comment:
no need to specify these enums, because we already have a meaningful prefix FEATHERING
so in QML you will type - BeamTypes.FEATHERING_NONE

Reply:
But `enum class` requires this specification in C++. In QML both `enum` and `enum class` don't need it.